oreilly-scenarios/cockroach-with-friendpay/intro.md

```diff
@@ -0,0 +1,8 @@
+![CockroachDB Logo](./assets/CockroachDB_logo.png)
+
+In this tutorial, you will:
+
+1. Validate an installation of CockroachDB.
+2. Learn about the FriendPay application.
```

Comment:
Change bullet 2 to something like "Run the FriendPay application, a lightweight toy payment app."